doc/source/ray-core/placement-group.rst

```diff
@@ -146,28 +151,20 @@ Strategy types
 
 Ray currently supports the following placement group strategies:
 
-**STRICT_PACK**
-
-All bundles must be placed into a single node on the cluster.
+**PACK**: All provided bundles are packed onto a single node on a best-effort basis.
 
-**PACK**
+**STRICT_PACK**: All bundles must be placed into a single node on the cluster.
+If strict packing is not feasible, bundles can be placed on multiple `PACK` nodes.
 
-All provided bundles are packed onto a single node on a best-effort basis.
-If strict packing is not feasible (i.e., some bundles do not fit on the node), bundles can be placed onto other nodes nodes.
+**SPREAD**: Each bundle will be spread onto separate nodes on a best effort basis.
 
-**STRICT_SPREAD**
+**STRICT_SPREAD**: Each bundle must be scheduled in a separate node.
+If strict spreading is not feasible, bundles can be placed on overlapping `SPREAD` nodes.
```

Comment:
Moved some wording around. I'd like a technical review to make sure this still makes sense.